pyvista/examples/downloads.py

```diff
@@ -8185,3 +8185,38 @@ def _full_head_files_func():  # pragma: no cover
 
 
 _dataset_full_head = _MultiFileDownloadableDatasetLoader(_full_head_files_func)
+
+
+def download_nek5000(load=True):
+    """Download 2D nek5000 data example.
+
+    .. versionadded:: 0.45.0
+
+    Parameters
+    ----------
+    load : bool, optional
+        Load the dataset after downloading it when ``True``.  Set this
+        to ``False`` and only the filename will be returned.
+
+    Returns
+    -------
+    pyvista.Nek5000Reader | str
+        DataSet or filename depending on ``load``.
+
+    Examples
+    --------
+    >>> from pyvista import examples
+    >>> dataset = examples.download_nek5000()
+    >>> dataset.plot()
+
```

Comment:
This will add a cross-ref to the dataset's entry in the gallery

```suggestion

    .. seealso::
    
        :ref:`Nek5000 Dataset <nek5000_dataset>`
            See this dataset in the Dataset Gallery for more info.

```